2200022/3499.py

```diff
@@ -0,0 +1,16 @@
+# 3499 퍼펙트 셔플
+
+T = int(input())
+for tc in range(1, T+1):
+    n = int(input())
+    cards = list(input().split())
+    result = []
+    l = 0
+    r = (n+1)//2
+    for i in range(n//2):
+        result.append(cards[l])
+        result.append(cards[r])
+        l, r = l+1, r+1
+    if n%2 == 0:
+        result.append(cards[n//2])
+    print(f'#{tc}',' '.join(result))
```

Comment:
깔끔한 풀이네요! 👍
```suggestion
    if n%2 == 1:
        result.append(cards[n//2])
    print(f'#{tc}',' '.join(result))
```
n이 짝수일 때가 아닌, 홀수일 때 하나의 원소를 더 추가하도록 수정해주세요!